rules/api/rules.py

```diff
@@ -251,3 +253,19 @@ def delete(self, request, *args, **kwargs):
             DELETE /api/proxy/rules/app/ HTTP/1.1
         """
         return super(RuleDetailAPIView, self).delete(request, *args, **kwargs)
+
+
+class UserEngagementAPIView(ListAPIView):
+
+    serializer_class = UserEngagementSerializer
+
+    def get_queryset(self):
+        User = get_user_model()
+        return User.objects.order_by('-last_login').prefetch_related('engagements')
+
+
+class EngagementAPIView(GenericAPIView):
+
+    def get(self, request, *args, **kwargs):
+        values = Engagement.objects.values('slug').annotate(count=Count('slug'))
+        return Response(values)
```

Comment:
Define a `serializer_class` and use it here otherwise the API doc is not generated correctly.